addins/shortcut-sample-with-taskpane-CF/home.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta http-equiv="Expires" content="0" />
+    <title>Shortcut Sample</title>
+    <script src="https://appsforoffice.edog.officeapps.live.com/lib/beta/hosted/office.js" type="text/javascript"></script>
```

Comment:
I think we don't need this line if we use the bundle.js.

Reply:
removed the bundle.js.